libraries/chain/controller.cpp

```diff
@@ -570,6 +570,9 @@ struct controller_impl {
          shutdown();
       }
 
+      if (conf.prune_config && conf.prune_config->prune_blocks == 0) {
+         blog.remove();
+      }
```

Comment:
Maybe it is fine to blow away the blocklog in these cases, but I wonder if we should require something like --delete-all to be passed for that to happen automatically. I would hate to accidently delete my 3TB blocklog. The config description does indicate it deletes the blocklog when done, so I guess this is ok.

Reply:
Doesn't seem any more dangerous than accidentally doing an existing passively allowed `block-log-retain-blocks = 1`. Not sure where the threshold should be for a `--i-really-mean-it` like option

Reply:
The description does indicate it deletes the blocklog, so that is good enough for me at least.